hack/local-up-cluster.sh

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -x
```

Comment:
@dims: this was probably added for debugging and shouldn't have been committed?

Reply:
Also have doubts about it, `set -x` outputs too much information.

Reply:
still need it. we are debugging the `docker stop` (we added a timeout)